### ISSUE TITLE
Accessibility: Restore Alt-N accelerator

### ISSUE THREAD

**Issue by ronys** [pwsafe/pwsafe#1892]
In the Basic tab of the add/edit dialog, the switch to subtabs for Notes and Custom Fields broke the Alt-N accelerator functionality. This should be restored Alt-N (or whatever is defined as the Notes accelerator key in the current language) should switch the keyboard focus to the Notes tab.
